frontend/express/public/javascripts/countly/countly.helpers.js

```diff
@@ -321,6 +321,16 @@
         countlyCommon.dispatchNotificationToast(payload);
     };
 
+    CountlyHelpers.goTo = function(options) {
+        app.backlinkRoute = options.from;
+        window.location.hash = options.url;
+    };
+
+    CountlyHelpers.getBacklinkRoute = function() {
+        var route = app.backlinkRoute;
+        app.backlinkRoute = null;
```

Comment:
Unfortunately, due to view re-renders I was not able to keep the backlink route reactive (add it in the global store) since we remove it after we read it. Hence, I decided to save it outside of the store.